src/tests/Calculator.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+// eslint-disable-next-line
+import Calculator from '../Component/Calculator';
```

Comment:
- [ ] I ran this test and I got this error seen in the image below. This is because the folder `comment` is supposed to have a lowercase c at the beginning. It should be like this `../component/operate`
![image](https://user-images.githubusercontent.com/45460192/228631364-f226dfb6-fa6f-475a-83f0-246f6486ad7b.png)